bin/generate-gutenboarding-design-thumbnails.js

```diff
@@ -37,9 +38,44 @@ const getDesignUrl = ( design ) => {
 	} );
 };
 
+/*
+A very fragile regex to grab the availableDesigns objects fom the file output string
+*/
+const designsFileContents = fs.readFileSync(
+	'./client/landing/gutenboarding/available-designs.ts',
+	'utf8'
+);
+// get everything in between setting the const and the next `;`
+const regex = /^(const availableDesigns: Readonly< AvailableDesigns > = )([^;]*)(;)$/m;
+const designsChunk = designsFileContents.match( regex );
+let featuredDesignsObject = [];
+
+try {
+	const trimmedString = designsChunk[ 2 ]
+		.toString()
+		.split( '\n' )
+		.map( ( line ) => line.trim() )
+		.filter( Boolean )
+		.join( '' );
+	featuredDesignsObject = eval( '(' + trimmedString + ')' ).featured;
```

Comment:
Clever regex use here! To avoid having to `eval` the string, would it be worth it to move the list of availableDesigns out of the `ts` file and into a `json` file that we can import from `available-designs.ts`, and use `JSON.parse` here? Alternatively, I just read about this approach to avoiding `eval` and [using window.Function()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval#Never_use_eval!) instead.

Reply:
Good point about going back to json file! :-) I was hoping we wouldn't need to run this script more than a few times and that we'd work on mshots instead of improving this script. 🙃 

https://github.com/Automattic/wp-calypso/pull/40632#discussion_r402218015

Reply:
> to avoid having to eval the string, would it be worth it to move the list of availableDesigns out of the ts file and into a json file that we can import from available-designs.ts

Totally worth it. And it will make this rush job looks less, well, like a rush job. 👍